### PR TITLE
Tag LRSLib.jl v0.0.3

### PR DESCRIPTION
**Pull request by blegat**
Repository: [JuliaPolyhedra/LRSLib.jl](https://github.com/JuliaPolyhedra/LRSLib.jl)
Release: [v0.0.3](https://github.com/JuliaPolyhedra/LRSLib.jl/releases/tag/v0.0.3)
Diff: [vs v0.0.2](https://github.com/JuliaPolyhedra/LRSLib.jl/compare/3f0caf4b378b7951ee76d91d436ee958d1d4840a...810919af3bbd56ac7f92a631902bff14d6dd7fa3)
`requires` vs v0.0.2:

```
--- 0.0.2/requires
+++ 0.0.3/requires
@@ -1,4 +1,4 @@
 julia 0.5
 BinDeps
-Polyhedra 0.1.2
+Polyhedra 0.1.4 0.2
 @osx Homebrew
```

### CODE REVIEW

File: LRSLib/versions/0.0.3/requires [hunk 1]
```diff
@@ -0,0 +1,4 @@
+julia 0.5
+BinDeps
+Polyhedra 0.1.4 0.2
```
Comment (tkelman) [inline LRSLib/versions/0.0.3/requires]: wasn't the api change in 0.1.3 here?

Comment (tkelman) [inline LRSLib/versions/0.0.3/requires]: https://github.com/JuliaPolyhedra/LRSLib.jl/compare/3f0caf4b378b7951ee76d91d436ee958d1d4840a...810919af3bbd56ac7f92a631902bff14d6dd7fa3 only changes getlibraryfor?

Comment (blegat) [inline LRSLib/versions/0.0.3/requires]: Yes, that is the only change needed from Polyhedra v0.1.2 to v0.1.4 for LRSLib :)

Comment (tkelman) [inline LRSLib/versions/0.0.3/requires]: and it was changed in Polyhedra 0.1 3, not 0.1.4? https://github.com/JuliaPolyhedra/Polyhedra.jl/commit/7b0b56bf72ca00380cb5b355d30179a925921fd5

Comment (blegat) [inline LRSLib/versions/0.0.3/requires]: Indeed ! Thank you for pointing this out, this should be fixed now :)

Comment (tkelman) [inline LRSLib/versions/0.0.3/requires]: don't forget to make the corresponding change at https://github.com/JuliaPolyhedra/LRSLib.jl/blob/master/REQUIRE before the next tag

Comment (blegat) [inline LRSLib/versions/0.0.3/requires]: Oops, indeed. Have updated the REQUIRE on master, the tag and the sha :)

**Pull request merged**